packages/@n8n/config/src/configs/tags.config.ts

```diff
@@ -0,0 +1,11 @@
+import { Config, Env } from '../decorators';
+
+@Config
+export class TagsConfig {
+	/*
+		Disable workflow tags
+	*/
+
+	@Env('N8N_WORKFLOW_TAGS_DISABLED')
+	workflowTagsDisabled: boolean = false;
```

Comment:
Maybe we call the property `disabled` so we can do `tags.disabled`. Or move this to the workflow configuration instead? 🤔

Reply:
yeah. let's do that.